### ISSUE TITLE
Report of incorrect a11y guidance for "Make Screen Reader Navigation Easier with the header Landmark"

### ISSUE THREAD

**Issue by ArielLeslie** [freeCodeCamp/freeCodeCamp#45091]
One of the forum users ([Marconiusiii](https://forum.freecodecamp.org/u/Marconiusiii)) has [raised an issue](https://forum.freecodecamp.org/t/accessibility-information-is-inaccurate/496713) about this accessibility lesson. Here are their remarks

This article/lesson is teaching a bad a11y practice. h1 elements do not belong in the header region since they are usually the first targeted element by screen reader users. Once I jump to the h1, I am expecting there to be page content ready to go when I start navigating normally element by element down the page. Putting the h1 in the header defeats the purpose of the h1 and the landmark since only repeated elements and object belong in the header, such as logos, navigation regions, search boxes, etc. The h1 will change with each page change, and navigating down from it should move me immediately into the page content, not contend with anything else in the header region.

Skip to content links generally aim for either the main region id or the h1 id, and if I land on an h2 in the main content, I feel like I’ve missed something and try navigating backwards to find the h1.

Please revise the practices being taught here and ensure that coders and designers don’t put their h1s incorrectly into header regions.

```html
<body>
<header>
<!-- h1 elements do not belong here! -->
  <h1>Training with Camper Cat</h1>
</header>

<main>
<!-- h1 elements belong right here! -->

  <section id="stealth">
    <h2>Stealth &amp; Agility Training</h2>
    <article><h3>Climb foliage quickly using a minimum spanning tree approach</h3></article>
    <article><h3>No training is NP-complete without parkour</h3></article>
  </section>
```
## Affected page

[Make Screen Reader Navigation Easier with the header Landmark](https://www.freecodecamp.org/learn/responsive-web-design/applied-accessibility/make-screen-reader-navigation-easier-with-the-header-landmark)

**Comment by ShaunSHamilton** [comment]
Thank you, for taking the time to open this issue.

**My suggestion**

We are not making large changes to the current curriculum, in favour of the new curriculum. Seeing as we do plan to sunset the old curriculum, I suggest we focus on any improvements to the _now_ Beta section of Responsive Web Design.

That is, in the _Accessibility Quiz_ project, move the location of the `h1`, and teach its importance. When I was designing that project, I specifically researched this very topic, but, as it was not against the _Web Accessibility Guidelines_, I thought little of it.

For the other practice projects, we should just ensure they follow the _best-practice_, with no specific need to teach anything extra - the point of the Accessibility practice project.

**Comment by bbsmooth** [comment]
Just realized we have this same issue in the new curriculum:

[Learn HTML by Building a Cat Photo App - Step 5](https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-html-by-building-a-cat-photo-app/step-5)

While it is not strictly an accessibility violation to have the `<h1>` outside of `<main>` it is a best practice to place it at the beginning of `<main>`. Also, it is a best practice to have all content inside of a landmark. The `<main>` element is a landmark, so we would fulfill this BP as well by putting the `<h1>` inside `<main>`.

**Comment by moT01** [comment]
I don't think fixing the legacy challenges is all that important, we can leave those if we want but we should probably fix up the new ones.

**Comment by Sboonny** [comment]
If you would like to fix this issue, please make sure you read [our guidelines for contributing](https://contribute.freecodecamp.org/#/), we prioritize contributors following the instructions in our guides. Join us in [our chat room](https://discord.gg/PRyKn3Vbay) or [the forum](https://forum.freecodecamp.org/c/contributors/3) if you need help contributing, our moderators will guide you through this.

**Comment by ShaunSHamilton** [comment]
> On a side note, when you move the `header` element inside the `main` element then it loses its banner landmark status and thus basically just becomes a `div` as far as accessibility is concerned. That's fine here, there is no need for a banner landmark in this simple menu.

Does this not defeat the purpose of a `header`? Should we scrap the `header` all-together from this project?
